src/main/kotlin/no/nav/tilleggsstonader/sak/migrering/routing/SøknadRoutingController.kt

```diff
@@ -12,18 +14,39 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping(path = ["/api/ekstern/routing-soknad"])
+@RequestMapping(path = ["/api/ekstern"])
 @ProtectedWithClaims(issuer = "azuread")
 class SøknadRoutingController(
     private val søknadRoutingService: SøknadRoutingService,
+    private val fagsakService: FagsakService,
 ) {
 
-    @PostMapping
+    @PostMapping("routing-soknad")
     @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
     fun sjekkRoutingForPerson(@RequestBody request: IdentStønadstype): SøknadRoutingResponse {
         feilHvisIkke(SikkerhetContext.kallKommerFra(EksternApplikasjon.SOKNAD_API), HttpStatus.UNAUTHORIZED) {
             "Kallet utføres ikke av en autorisert klient"
         }
         return søknadRoutingService.sjekkRoutingForPerson(request)
     }
+
+    @PostMapping("harBehandling")
+    @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
+    fun hentBehandlingStatusForPersonMedStønadstype(@RequestBody identStønadstype: IdentStønadstype): Boolean {
+        val behandlinger = fagsakService.hentBehandlingerForPersonOgStønadstype(
+            identStønadstype.ident,
+            identStønadstype.stønadstype,
+        )
+        if (behandlinger.isNotEmpty()) {
+            val validStatuses = listOf(
+                BehandlingStatus.OPPRETTET,
+                BehandlingStatus.UTREDES,
+                BehandlingStatus.FATTER_VEDTAK,
+                BehandlingStatus.SATT_PÅ_VENT,
+            )
+            val behandlingStatus = behandlinger.map { it.status }
+            return behandlingStatus.any { it in validStatuses }
+        }
```

Comment:
Hadde vært fint hvis dette bre trukket ut i egen metode i et object, og at det enkelt kunde testes

Reply:
Lagt ny service for dette.